homeassistant/components/matter/strings.json

```
@@ -119,6 +119,9 @@
       },
       "on_off_transition_time": {
         "name": "On/Off transition time"
+      },
+      "altitude": {
+        "name": "Altitude above Sea Level"  
```

It seems there are unnecessary spaces here:

```suggestion
        "name": "Altitude above Sea Level"
```